src/main/java/seedu/address/model/Model.java

```diff
@@ -85,6 +85,11 @@ public interface Model {
      */
     void setPerson(Person target, Person editedPerson);
 
+    /**
+     * Returns a {@code String} containing all tags in printable format.
+     */
+    String getAllTagsInPrintableFormat();
```

Comment:
How about the name `prettyPrintTags()`?

Reply:
@kelvinou01 it will still need to start with a verb. so maybe `printTagsPrettily`?